contrib/libvpl/module.defs

```diff
@@ -0,0 +1,50 @@
+$(eval $(call import.MODULE.defs,LIBVPL,libvpl))
+$(eval $(call import.CONTRIB.defs,LIBVPL))
+
+LIBVPL.FETCH.url     = https://github.com/oneapi-src/oneVPL/archive/refs/tags/v2021.6.0.tar.gz
```

Comment:
You can add the HandBrake secondary URL here now. It should be uploaded to our HandBrake-contribs repo

Reply:
done